src/fmu/dataio/export/rms/inplace_volumes.py

```diff
@@ -24,7 +25,17 @@
 _logger: Final = null_logger(__name__)
 
 
-_TABLE_INDEX_COLUMNS: Final = ("ZONE", "REGION", "FACIES", "LICENCE")
+_TABLE_INDEX_COLUMNS: Final = ("ZONE", "REGION", "FACIES", "LICENSE")
```

Comment:
typo in `LICENSE` fixed

Reply:
I think this was not your typo:

https://github.com/equinor/fmu-dataio/blob/61d17d4b54dddcd901a9ea6ff56485a6b3c0d95a/src/fmu/dataio/_definitions.py#L73

We probably want to change this too? Doesn't need to be in your PR

Reply:
👍 I'll make an issue of it, and also state that we should add `FLUID` in there